### ISSUE TITLE
Raise/assume status code to be 200 if status code is not specified?

### ISSUE THREAD

**Issue by ivan-kolmychek** [pact-foundation/pact-mock_service#22]
I'm not sure where to post it - I encountered it when used pact + pact-mock_service, so, I've decided to post it here.

When you accidentaly forget to specify `status` response option you can get strange result with `Net::HTTP`:

```
Net::HTTPBadResponse:
       wrong status line: "HTTP/1.1 0  "
```

I see such possible solutions:
- raise an exception if status is not specified - it would be nice because error message can explicitly say "you forgot the status code" instead of assuming it as `0` 
- or assume it to be `200` - it's the most common expected response code IMO;

**Comment by bethesque** [comment]
I think an error would be best, assumptions can be dangerous.

**Comment by ivan-kolmychek** [comment]
@bethesque I agree - it's better to explicitly point to this error instead of assuming anything (0 or 200), but I just wanted to present other possible solution I've thought about anyway.

**Comment by YOU54F** [comment]
[pact mock server](https://github.com/pact-foundation/pact-core-mock-server) has been written in written in rust, avaialble as both a CLI and FFI interface for interop with client languages

If there are further issues, they should be raised in their respective repos
